paasta_tools/utils.py

```diff
@@ -1340,3 +1340,16 @@ def format_table(rows, min_spacing=2):
             expanded_rows.append(expanded_row)
 
     return [(' ' * min_spacing).join(r) for r in expanded_rows]
+
+
+def deep_merge_dictionaries(source, destination):
```

Comment:
This is fixed in the latest commit. If a value and a dictionary share the same key, whatever is in the per-instance config will override the per-service config.